compiler/passes/convert-uast.cpp

```diff
@@ -47,60 +48,8 @@ struct Converter {
   chpl::Context* context = nullptr;
   bool inTupleDecl = false;
 
-  // Cache strings for special identifiers we might compare against.
```

Comment:
Thanks for updating convert-uast. Would you be willing to do some grepping in other non-test code for `== "` and `!= "` to find other uses? In particular I would appreciate if you can update the code in resolution-queries.cpp.

Reply:
Updated resolution-queries. I commented out UniqueString operator== and != on const char* and found a few other places that I've updated. Did not do any of the test code